index.js

```diff
@@ -226,7 +231,16 @@ const compile = function(schema, root, reporter, opts, scope) {
       unprocessed.delete(property)
     }
 
+    if (node === root && typeof node.$schema === 'string') {
+      if (!schemaVersions.includes(node.$schema)) throw new Error('Unexpected schema version')
+      consume('$schema') // meta-only
+    }
+
     if (typeof node.description === 'string') consume('description') // unused, meta-only
+    if (typeof node.title === 'string') consume('title') // unused, meta-only
+    if (typeof node.$comment === 'string') consume('$comment') // unused, meta-only
+    if (Array.isArray(node.examples)) consume('examples') // unused, meta-only
```

Comment:
This array could contain any kind of value, regexes or objects or giant strings, is that a concern? 

Perhaps we should delete meta-properties like this during validation.

Reply:
That's `.examples` in the _schema_ object. We don't mutate it.

Reply:
Ah right, doh!